### DISCUSSION TITLE
Skin element inconsistencies when using Barrel Roll mod

### DISCUSSION THREAD

**Discussion by NatureHog** [ppy/osu#18192] [Q&A]
I don't know if this was intentional, (that's why I'm opening a discussion instead of an issue, ) but I just tried using the Barrel Roll mod (with a "legacy" skin) and made a few observations:

- The Combo numbers inside the actual hitcircle always shows the correct way up
- The hitbursts do not, instead they always rotate the same way as the playfield
- Spinner elements, like the rpm at the bottom, also rotate together with the playfield

I don't know if it's just me being picky, but to me this feels somewhat inconsistent, as the hitcircles always seems right-side-up, but the spinner doesn't...

I'm afraid I can't explain it better, so here's a video (on OneDrive) to demonstrate:
https://1drv.ms/v/s!AqwZ15G5dYM5gvhhnUuwvnZUTNWTzQ?e=JcmKzR

**Comment by smoogipoo** [comment]
The combo numbers not rotating was a conscious decision. I could probably agree with UI elements (such as spinner overlays) in general not rotating.

It would probably be a matter of rotating the hitobject positions rather than rotating the playfield in entirety, to fix.

**Reply by NatureHog** [reply]
I don't really care if a) everything is rotating along with BR, or b) if everything always faces the correct way, but it seems inconsistent the way it is now...
